### DISCUSSION TITLE
v1.9.2-rc3

### DISCUSSION THREAD

**Discussion by olljanat** [burmilla/os#101] [General]
# About this release
This release is marked as release candicate because of major version change on Docker and Buildroot.

It is still possible to upgrade this one from earlier releases with command:
```
sudo ros os upgrade --image burmilla/os:v1.9.2-rc3
```

## Release notes
- Linux 4.14.235
- Buildroot: 2021.02.2
- Docker docker-20.10.7 by default

<hr /><em>This discussion was created from the release <a href='https://github.com/burmilla/os/releases/tag/v1.9.2-rc3'>v1.9.2-rc3</a>.</em>

**Comment by ashutoshbbpandey** [comment]
When can we expect v1.9.2 release?

**Reply by olljanat** [reply]
I guess that we will release v1.9.2 with Docker 20.10.8 when that is available. However like mentioned above this one is marked as RC just to give users notification that they should test this one version extra carefully before upgrading their business critical servers to using it and report about any new bugs about they found. v1.9.2 can contains fixes to those issues only if we get those reports.
